************* Module pathlib
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1:0: C0302: Too many lines in module (1559/1000) (too-many-lines)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:16:0: C0103: Constant name "supports_symlinks" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:19:7: E1101: Module 'sys' has no 'getwindowsversion' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:22:8: C0103: Constant name "supports_symlinks" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:23:8: C0103: Constant name "_getfinalpathname" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:25:4: C0103: Constant name "nt" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:56:0: R0205: Class '_Flavour' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:61:20: E1101: Instance of '_Flavour' has no 'sep' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:65:14: E1101: Instance of '_Flavour' has no 'sep' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:66:17: E1101: Instance of '_Flavour' has no 'altsep' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:68:8: C0103: Variable name "it" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:74:29: E1101: Instance of '_Flavour' has no 'splitroot' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:76:20: C0103: Variable name "x" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:87:20: W0621: Redefining name 'part' from outer scope (line 69) (redefined-outer-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:92:30: E1101: Instance of '_Flavour' has no 'splitroot' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:63:4: R0912: Too many branches (14/12) (too-many-branches)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:101:4: R0913: Too many arguments (7/5) (too-many-arguments)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:110:30: E1101: Instance of '_Flavour' has no 'casefold' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:110:53: E1101: Instance of '_Flavour' has no 'casefold' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:170:20: R1705: Unnecessary "else" after "return" (no-else-return)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:184:4: C0103: Argument name "s" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:184:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:187:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:190:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:194:8: C0103: Variable name "s" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:199:12: R1705: Unnecessary "else" after "return" (no-else-return)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:205:24: C0103: Variable name "s" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:208:24: C0103: Variable name "s" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:217:4: C0103: Argument name "s" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:217:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:227:4: C0103: Argument name "s" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:246:8: R1705: Unnecessary "else" after "return" (no-else-return)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:243:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:300:12: R1705: Unnecessary "else" after "return" (no-else-return)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:292:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:307:4: C0103: Argument name "s" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:307:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:310:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:313:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:318:19: W0212: Access to a protected member _accessor of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:344:16: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:361:26: W0613: Unused argument 'parts' (unused-argument)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:361:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:364:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:375:16: C0415: Import outside toplevel (pwd) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:378:12: C0415: Import outside toplevel (pwd) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:382:16: W0707: Consider explicitly re-raising using the 'from' keyword (raise-missing-from)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:370:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:390:0: R0903: Too few public methods (0/2) (too-few-public-methods)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:410:17: E1101: Module 'os' has no 'lchmod' member; maybe 'chmod'? (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:423:8: W0211: Static method with 'self' as first argument (bad-staticmethod-argument)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:436:12: C0103: Argument name "a" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:436:12: C0103: Argument name "b" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:436:12: E0213: Method should have "self" as first argument (no-self-argument)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:441:8: C0103: Argument name "a" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:441:8: C0103: Argument name "b" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:441:26: W0613: Unused argument 'target_is_directory' (unused-argument)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:447:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:395:0: R0903: Too few public methods (1/2) (too-few-public-methods)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:494:18: W0212: Access to a protected member _accessor of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:497:15: E1101: Instance of '_Selector' has no '_select_from' member; maybe 'select_from'? (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:475:0: R0903: Too few public methods (1/2) (too-few-public-methods)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:502:40: W0613: Unused argument 'is_dir' (unused-argument)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:502:48: W0613: Unused argument 'exists' (unused-argument)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:502:56: W0613: Unused argument 'scandir' (unused-argument)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:502:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:500:0: R0903: Too few public methods (0/2) (too-few-public-methods)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:514:19: W0212: Access to a protected member _make_child_relpath of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:516:20: C0103: Variable name "p" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:516:25: W0212: Access to a protected member _select_from of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:506:0: R0903: Too few public methods (1/2) (too-few-public-methods)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:535:16: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:541:31: W0212: Access to a protected member _make_child_relpath of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:542:28: C0103: Variable name "p" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:542:33: W0212: Access to a protected member _select_from of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:522:0: R0903: Too few public methods (1/2) (too-few-public-methods)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:551:23: W0613: Unused argument 'pat' (unused-argument)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:562:16: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:566:27: W0212: Access to a protected member _make_child_relpath of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:567:24: C0103: Variable name "p" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:576:35: W0212: Access to a protected member _select_from of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:578:24: C0103: Variable name "p" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:549:0: R0903: Too few public methods (1/2) (too-few-public-methods)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:605:8: R1705: Unnecessary "else" after "return" (no-else-return)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:620:0: R0205: Class 'PurePath' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:641:12: W0642: Invalid assignment to cls in method (self-cls-assignment)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:654:12: C0103: Variable name "a" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:656:25: W0212: Access to a protected member _parts of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:658:16: C0103: Variable name "a" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:667:15: E1101: Class 'PurePath' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:674:27: W0212: Access to a protected member _parse_args of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:675:8: W0212: Access to a protected member _drv of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:676:8: W0212: Access to a protected member _root of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:677:8: W0212: Access to a protected member _parts of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:679:12: W0212: Access to a protected member _init of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:685:8: W0212: Access to a protected member _drv of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:686:8: W0212: Access to a protected member _root of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:687:8: W0212: Access to a protected member _parts of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:689:12: W0212: Access to a protected member _init of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:694:8: R1705: Unnecessary "else" after "return" (no-else-return)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:695:32: E1101: Class 'PurePath' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:697:19: E1101: Class 'PurePath' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:705:27: E1101: Instance of 'PurePath' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:725:8: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:725:12: E1101: Instance of 'PurePath' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:740:15: E1101: Instance of 'PurePath' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:748:34: E1101: Instance of 'PurePath' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:754:49: E1101: Instance of 'PurePath' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:764:46: E1101: Instance of 'PurePath' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:769:46: E1101: Instance of 'PurePath' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:774:46: E1101: Instance of 'PurePath' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:779:46: E1101: Instance of 'PurePath' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:812:8: R1705: Unnecessary "else" after "return" (no-else-return)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:835:8: R1705: Unnecessary "else" after "return" (no-else-return)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:844:27: E1101: Instance of 'PurePath' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:845:37: E1101: Instance of 'PurePath' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:845:56: E1101: Instance of 'PurePath' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:856:8: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:856:12: E1101: Instance of 'PurePath' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:895:8: C0103: Variable name "n" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:896:8: C0103: Variable name "cf" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:896:13: E1101: Instance of 'PurePath' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:956:19: E1101: Instance of 'PurePath' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:961:15: E1101: Instance of 'PurePath' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:967:8: C0103: Variable name "cf" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:967:13: E1101: Instance of 'PurePath' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:969:31: E1101: Instance of 'PurePath' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:685:8: W0201: Attribute '_drv' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:686:8: W0201: Attribute '_root' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:687:8: W0201: Attribute '_parts' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:715:12: W0201: Attribute '_str' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:748:12: W0201: Attribute '_cached_cparts' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:760:12: W0201: Attribute '_hash' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:913:12: W0201: Attribute '_pparts' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1032:12: W0642: Invalid assignment to cls in method (self-cls-assignment)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1030:0: W0613: Unused argument 'kwargs' (unused-argument)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1040:4: W0221: Parameters differ from overridden '_init' method (arguments-differ)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1046:29: W0212: Access to a protected member _accessor of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1061:4: C0103: Argument name "t" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1061:4: C0103: Argument name "v" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1061:4: C0103: Argument name "tb" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1064:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1067:22: W0613: Unused argument 'name' (unused-argument)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1094:19: E1101: Instance of 'Path' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1094:19: W0212: Access to a protected member _flavour of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1100:8: C0103: Variable name "st" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1127:35: E1101: Instance of 'Path' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1130:56: E1101: Instance of 'Path' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1131:12: C0103: Variable name "p" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1139:35: E1101: Instance of 'Path' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1142:66: E1101: Instance of 'Path' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1143:12: C0103: Variable name "p" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1161:8: W0212: Access to a protected member _init of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1172:8: C0103: Variable name "s" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1172:12: E1101: Instance of 'Path' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1177:12: C0103: Variable name "s" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1179:17: E1101: Instance of 'Path' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1181:8: W0212: Access to a protected member _init of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1195:8: C0415: Import outside toplevel (pwd) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1202:8: C0415: Import outside toplevel (grp) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1205:4: R0913: Too many arguments (6/5) (too-many-arguments)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1220:37: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1227:70: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1236:37: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1246:70: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1269:8: C0103: Variable name "fd" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1384:8: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1399:8: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1416:8: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1453:8: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1468:8: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1484:8: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1500:8: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1516:8: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1532:22: E1101: Instance of 'Path' has no '_flavour' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1062:8: W0201: Attribute '_closed' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:1016:0: R0904: Too many public methods (37/20) (too-many-public-methods)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:10:0: C0411: standard import "from errno import EINVAL, ENOENT, ENOTDIR, EBADF, ELOOP" should be placed before "from _collections_abc import Sequence" (wrong-import-order)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:11:0: C0411: standard import "from operator import attrgetter" should be placed before "from _collections_abc import Sequence" (wrong-import-order)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:12:0: C0411: standard import "from stat import S_ISDIR, S_ISLNK, S_ISREG, S_ISSOCK, S_ISBLK, S_ISCHR, S_ISFIFO" should be placed before "from _collections_abc import Sequence" (wrong-import-order)
/log/firmware/firmware/usr/lib/python3.8/pathlib.py:13:0: C0411: standard import "from urllib.parse import quote_from_bytes as urlquote_from_bytes" should be placed before "from _collections_abc import Sequence" (wrong-import-order)

-----------------------------------
Your code has been rated at 6.24/10

